src/components/BlogSection/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,44 @@
+import Button from '../../UI/Button';
+import PropTypes from 'prop-types';
+
+const BlogPost = ({ title, author, date, content, imageUrl, id, onClick }) => {
+  const shortTitle = title.length >= 35 ? title.slice(0, 35) + ' ...' : title;
+  const contentShort = content.slice(0, 100) + ' ...';
+
+  const showPostDetailHandler = () => {
+    onClick(id);
+  };
+
+  return (
+    <div className="post" onClick={showPostDetailHandler}>
```

Comment:
Try to avoid `onClick` handlers on _non clickable_ elements. Click should be handled only on element such as button a - this is because it has impact on a11y

Reply:
So I change tag <div> into <button /> as post container